cdc/owner.go

```diff
@@ -114,32 +117,52 @@ func (c *changeFeed) updateProcessorInfos(processInfos model.ProcessorsInfos) {
 	c.processorInfos = processInfos
 }
 
+func (c *changeFeed) addSchema(schemaID uint64) {
+	c.schemas[schemaID] = make(map[uint64]struct{})
```

Comment:
Add a check here so that when `addSchema` is called in the wrong place it doesn't silently empty the existing tables?

Reply:
added, PTAL